myia/compile/backends/python/optimizer.py

```diff
@@ -36,15 +36,15 @@ def optimize(self, directed_graphs):
         todo = list(directed_graphs)
         seen = set()
         while todo:
-            dg = todo.pop(0)
+            dg = todo.pop(0)  # type: DirectedGraph
```

Comment:
Might as well use an annotation:

```suggestion
            dg: DirectedGraph = todo.pop(0)
```

Reply:
Ok!